### ISSUE TITLE
Authorization not working after contacting decive.

### ISSUE THREAD

**Issue by skirners1** [playgameservices/play-games-plugin-for-unity#467]
Hello!

My authorization process comes to an end after choosing GPG (Google Play Games) account in the pop up menu, while in game and trying to login to GPG.
I have struggled with this one for some time now and searched the web inside and out for some answears with no luck..

I have registered as beta in GPG services and in Gamed developer, I have tried various keystores so I know I have the right SHA1 (otherwise GPG services gives me a warning), I do activate GPG before I authenticate. I have tried the minimal project but it doesn't work..

Did anyone solve this problem? Is there any other easy way to this plugin, to get a unified highscore between various mobile OS?

Thank you very much for your concern.

**Comment by claywilkinson** [comment]
Can you post a log file of what is happening when you are logging on?

**Comment by skirners1** [comment]
Hey!
Thanks for your concern! I'm only logging stuff directly to the UI. So only my own messages. I'll learn logcat and such when that day comes.

I solved the problem. I just had to remake the apk with a new keystore and also a whole new play games service. This wasn't the case the first and the second and the third time i tried this. However this time I follwed this (https://www.youtube.com/watch?v=XmHXl-UFTqM) guide to the millimeter, and it worked. Maybe it had something to do with the SHA1 and the keystore, or the fact that I published beta versions and gpg-service, or that I didn't have a publishversion uploaded, or maybe I did something in the wrong order. Which ever factor it was it's now working.

Have a great day!!
